src/commands/compile.rs

```diff
@@ -0,0 +1,39 @@
+/*
+ * Compiler-Bot: compiler bot for Unofficial.CSE
+ * Copyright (C) 2025  Unofficial.CSE contributors
+ *
+ * Compiler-Bot is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Compiler-Bot is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Compiler-Bot.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+use crate::{Context, Error};
+
+/// Compile and run code
+#[poise::command(prefix_command, slash_command)]
```

Comment:
We won't be supporting the `slash_command` form of the compilation command for now, so don't register it as one.

```suggestion
#[poise::command(prefix_command)]
```